From dbf7e56888a6b2ad09118a267d3edbf55f259c8f Mon Sep 17 00:00:00 2001
From: Michael Polenchuk <mpolenchuk@mirantis.com>
Date: Wed, 26 Oct 2016 15:26:11 +0300
Subject: [PATCH] Disable public ping_checker resource by default

Avoid public vip failover in the middle of deployment
by disabling ping_checker resource by default.

Change-Id: I666fbbbe98d000c66f97cf4751f304a2da9aad67
---
 .../manifests/virtual_ips/public_vip_ping.pp       |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deployment/puppet/osnailyfacter/manifests/virtual_ips/public_vip_ping.pp b/deployment/puppet/osnailyfacter/manifests/virtual_ips/public_vip_ping.pp
index 49d9c42..c7f6cbd 100644
--- a/deployment/puppet/osnailyfacter/manifests/virtual_ips/public_vip_ping.pp
+++ b/deployment/puppet/osnailyfacter/manifests/virtual_ips/public_vip_ping.pp
@@ -4,7 +4,7 @@ class osnailyfacter::virtual_ips::public_vip_ping {
 
   $network_scheme = hiera_hash('network_scheme', {})
   prepare_network_config($network_scheme)
-  $run_ping_checker = hiera('run_ping_checker', true)
+  $run_ping_checker = hiera('run_ping_checker', false)
   $public_iface = get_network_role_property('public/vip', 'interface')
   $ping_host_list = $network_scheme['endpoints'][$public_iface]['gateway']
 
-- 
1.7.9.5

